src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -872,6 +917,9 @@ subroutine ocn_vmix_cvmix_init(domain,err)!{{{
       call mpas_pool_get_config(ocnConfigs, 'config_cvmix_convective_diffusion', config_cvmix_convective_diffusion)
       call mpas_pool_get_config(ocnConfigs, 'config_cvmix_convective_viscosity', config_cvmix_convective_viscosity)
       call mpas_pool_get_config(ocnConfigs, 'config_cvmix_kpp_use_enhanced_diff', config_cvmix_kpp_use_enhanced_diff)
+      call mpas_pool_get_config(ocnConfigs, 'config_cvmix_kpp_langmuir_mixing_opt', config_cvmix_kpp_langmuir_mixing_opt)
+      call mpas_pool_get_config(ocnConfigs, 'config_cvmix_kpp_langmuir_entrainment_opt', &
+                                config_cvmix_kpp_langmuir_entrainment_opt)
```

Comment:
please add a check below that user input values are acceptable values of your two character strings.  You can follow L1022 and following for an example check.

Reply:
@vanroekel Added.